ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -253,7 +253,7 @@ def create_torch_policy(
             self.seed,
             behavior_spec,
             self.trainer_settings,
-            condition_sigma_on_obs=False,  # Faster training for PPO
+            condition_sigma_on_obs=True,  # Faster training for PPO
```

Comment:
Curious why you switched this. Does it perform better? I know in the past we found this less stable.